src/main/java/ulcrs/controllers/TutorController.java

```diff
@@ -43,7 +43,7 @@ public boolean shouldSkipClass(Class<?> clazz) {
                         })
                         .setPrettyPrinting()
                         .create();
-            	return gson.toJson(tutors);
+                return gson.toJson(tutors);
             });
             get("/:id", this::getTutor, gson::toJson);

```

Comment:
Same as other comment, I'm not sure if this will work with the GsonFactory as you want all possible attributes here

Reply:
You wrote this block of code. Here are only format improvements

Reply:
What I was getting at is the GsonFactory dilemma, since this would exclude according to some annotations since `gson` in this case is made from the factory

Reply:
Got it. Now gson is the full Gson and exposeOnlyGson is the excluded version of Gson. So gson here would be fine.